### ISSUE TITLE
import returning Symbol, preventing inheritance

### ISSUE THREAD

**Issue by tylershuster** [jagi/meteor-astronomy#646]
I have a file, `user.js`, let's say it looks like:

```
import { Class } from 'meteor/jagi:astronomy';
const User = Class.create({...});
export default User;
```

I have a second file, `volunteer.js` which defines a type of user. It looks like this:

```
import User from '.';
User.inherit({...});
```

When I run Meteor, and I've verified that no dependencies other than Meteor and { Class } are loaded first, I get the message `TypeError: User.inherit is not a function at volunteer.js` and logging the imported `User` yields `{ [Symbol(__esModule)]: true }`.

Searching for this output with "esModule" has yielded nothing. Any idea why it might be trying to use the module before it's finished loading?

**Comment by tylershuster** [comment]
Let's pretend this never happened.

**Comment by johnsunam** [comment]
> Let's pretend this never happened.

Did you resolve this issue I am getting the same can you help me on this.

**Comment by tylershuster** [comment]
Wow, I'm sorry. That's a classic jerk move. I do not remember what changed. I haven't worked with Meteor since then, so I'm racking my brain. I took a look at the commit I made after this. One thing that wouldn't work in the example I made is that you'd have to rename `user.js` to `index.js` for the `import User from '.'` to work. Other than that, I can't see anything that would help you. The only thing I can guess is that I had an import statement elsewhere or a general code error.
